tests/Doctrine/Migrations/Tests/Metadata/Storage/TableMetadataStorageTest.php

```diff
@@ -355,4 +355,42 @@ public function testResetWithEmptySchema(): void
         );
         self::assertCount(0, $this->connection->fetchAllAssociative($sql));
     }
+
+    public function testGetSql(): void
+    {
+        $this->storage->ensureInitialized();
+
+        $result = new ExecutionResult(new Version('2230'), Direction::UP, new DateTimeImmutable('2010-01-05 10:30:21'));
+
+        $queries = $this->storage->getSql($result);
+
+        self::assertCount(2, $queries);
+        self::assertSame('-- Version 2230 update table metadata', $queries[0]->getStatement());
+        self::assertSame('INSERT INTO doctrine_migration_versions (version, executed_at, execution_time) VALUES (\'2230\', \'2010-01-05 10:30:21\', 0)', $queries[1]->getStatement());
```

Comment:
You can avoid the escaping by using double quotes. Also, this line is too long, please use linebreaks. Same below.